www/pfSense-pkg-vHosts/Makefile

```diff
@@ -0,0 +1,67 @@
+# Created by: Steve Thames <sthames42@gmail.com>
+# $FreeBSD$
+#*******************************************************************************
+# vHosts package makefile.
+# see: https://github.com/pfsense/FreeBSD-ports/blob/devel/Mk/bsd.port.mk
+# and: https://www.freebsd.org/doc/en/books/porters-handbook/
+# ------------------------------------------------------------------------------
+# Part of pfSense 2.3 and later (https://www.pfSense.org/).
+# Copyright (c) 2004-2016 Rubicon Communications, LLC (Netgate)
+#
+# Inspired by vhosts package originally written by Mark Crane.
+# Copyright (C) 2008 Mark J Crane
+# Copyright (C) 2015 ESF, LLC
+# Copyright (C) 2016 Softlife Consulting
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*******************************************************************************
```

Comment:
I don't want to be difficult but there is nothing in the style guide about makefiles.